FlashSpace/Settings/SettingsRepository.swift

```diff
@@ -74,13 +79,24 @@ final class SettingsRepository: ObservableObject {
     init() {
         encoder.outputFormatting = .prettyPrinted
         loadFromDisk()
+        runScriptOnBootstrapIfNeeded()
 
         debouncedUpdateSettings
             .debounce(for: .seconds(1), scheduler: DispatchQueue.main)
             .sink { [weak self] in self?.updateSettings() }
             .store(in: &cancellables)
     }
 
+    private func runScriptOnBootstrapIfNeeded() {
+        let script = runScriptOnBootstrap.trimmingCharacters(in: .whitespaces)
+        guard !script.isEmpty else { return }
+
+        let task = Process()
+        task.launchPath = "/bin/sh"
+        task.arguments = ["-c", script]
+        task.launch()
+    }
+
```

Comment:
Please move it to `Integrations.swift` and also add check if integrations are enabled (the same way as in `runIfNeeded`).

Please also rename `runIfNeeded` to `runOnActivateIfNeeded` and name your function `runOnAppLaunchIfNeeded`.

Reply:
I fixed it as you said,
But I used GCD because has initialization time issue.